test/mocks/delinkedCalculation.js

```diff
@@ -18,12 +19,15 @@ const mockDelinkedCalculation1 = {
   referenceAmount: 195963.84,
   totalProgressiveReduction: 299948.34,
   totalDelinkedPayment: 103984.50,
-  paymentAmountCalculated: 51992.25
+  paymentAmountCalculated: 51992.25,
+  datePublished: new Date(),
+  updated: new Date()
 }
 
 const mockDelinkedCalculation2 = {
   applicationId: 1810908,
   calculationId: 123456788,
+  calculationReference: 123456788,
```

Comment:
if we were to make the change to databases above then we have no need for a calculationReference here.